DjangoProject/DjangoProject/settings.py

```diff
@@ -128,3 +130,6 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 
+AUTH_USER_MODEL = 'users.User'
```

Comment:
only `AUTH_USER_MODEL` should be set here, it is required when default user model is changed